### ISSUE TITLE
seafile creates an shortcut/virtual drive in my computer, named "my library"

### ISSUE THREAD

**Issue by jarnix** [haiwen/seafile#1180]
and it should be removable from the client

**Comment by shoeper** [comment]
Afaik you told Seafile to do so. I don't have one and if I remember right the option for this shortcut can be created thorugh initial client setup.

**Comment by jarnix** [comment]
You're right, I remember clicking fast on the default values :(

This is intrusive so it's okay if I can remove it. I've been looking for how to remove this virtual drive, maybe not enough but I did not find how. So yes this is an option, but this option should be .. really optional ;)

Also, I removed the files in this folder, and now I just have a dead link in "my computer", so it's kind of a bug, that's why I reported it, would not have called it "ugly" if it was easy to remove and useful.

**Comment by p-schneider** [comment]
I found a way to manually get rid of this link by deleting the entry in the Windows registry (using regedit).

First I searched the registry (run->regedit) for seafile and found `[HKEY_CLASSES_ROOT\CLSID\{F817C393-A76E-435E-B6B1-485844BC9C2E}]`. Then I searched for `{F817C393-A76E-435E-B6B1-485844BC9C2E}` and found the reference in `[HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\MyComputer\Namespace\{F817C393-A76E-435E-B6B1-485844BC9C2E}]` (In my case it was the only key under Namespace.)
I'm not sure if the CLSID (GUID) would the same on every computer having seafile. If you want to get rid of "My Library" go to ``[HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\MyComputer\Namespace]` and delete the Key `{F817C393-A76E-435E-B6B1-485844BC9C2E}` (after making sure it is referencing seafile and not something else you might want to keep).
After that I didn't even have to log off / restart or kill the explorer to get it removed, F5 (refresh) in My Computer did the trick.

There is no need to also delete `[HKEY_CLASSES_ROOT\CLSID\{F817C393-A76E-435E-B6B1-485844BC9C2E}]` but on the other hand there should be no harm in deleting it.

I would like to see a Setting in Seafile to get rid of this thing. I don't like that kind of links in My Computer, especially not if it's putting itself before / next to my System Drives C, D etc..
(I see why the word "ugly" had been in the title of this issue.)
